docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -136,16 +129,18 @@ The minimal required permissions on LakeFS are:
   ]
 }
 ```
-### AWS Credentials
-Currently, `lakefs-sgc` does not provide an option to explicitly set AWS credentials. It relies on the hosting machine
-to be set up correctly, and reads the AWS credentials from the machine.
 
-This means, you should set up your machine however AWS expects you to set it. \
-For example, by following their guide on [configuring the AWS CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+### Credentials
+
+Standalone GC supports S3 and S3-compatible storage backends and relies on AWS credentials for authentication. To set up
+credentials, follow AWS guidelines, such as those outlined in [this guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+For details on how to pass credentials to `lakefs-sgc`, refer to the instructions in [How to Run Standalone GC](#how-to-run-standalone-gc).
```

Comment:
Consider being a bit more direct about configuring the docker container:
```suggestion
Standalone GC supports S3 and S3-compatible storage backends and relies on AWS credentials for authentication. To set up
credentials on the `lakefs-sgc` docker container, follow AWS guidelines, such as those outlined in [this guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
For details on how to pass credentials to `lakefs-sgc`, refer to the instructions in [How to Run Standalone GC](#how-to-run-standalone-gc).
```

Reply:
done